cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/autocomplete-config.js

```diff
@@ -8,8 +49,17 @@ function enableAutocomplete(config) {
     const inputElements = document.querySelectorAll('#aa-search-input');
     const contentSearchbarTab = $('#content-search-bar-button');
     const productSearchbarTab = $('#product-search-bar-button');
-    const contentTabPane = $('#content-search-results-pane')
-    const productTabPane = $('#product-search-results')
+    const contentTabPane = $('#content-search-results-pane');
+    const productTabPane = $('#product-search-results');
+    const recommendClient = config.recommendClient;
+
+    const RecommendConfig = {
+        productsIndex: algoliaData.productsIndex,
+        maxRecommendations: 5,
+        recommendClient: recommendClient,
+    };
+
+    fetchTrendingItems(RecommendConfig);
```

Comment:
This generate a network call on page load even before putting the focus on Autocomplete. Could we only do that on the first call to `getSources()`?